File: Leetcode/58-Length-of-Last-Word/LengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int res = 0;
        int an = 0;
        for (int i=0; i<s.length(); i++) {
            if (s.at(i) == ' ') {
                if (res != 0) {
                    an = res;
                }
                res = 0;
            } else {
                res++;
            }
        }
        if (res!=0) {
            return res;
        }
        return an;
    }
};

File: Leetcode/58-Length-of-Last-Word/LengthOfLastWord.go
func lengthOfLastWord(s string) int {
	res := 0
	an := 0
	for i := 0; i < len(s); i++ {
		if s[i] == ' ' {
			if res != 0 {
				an = res
			}
			res = 0
		} else {
			res++
		}
	}
	if res != 0 {
		return res
	}
	return an
}
